main.go

```diff
@@ -143,10 +145,11 @@ func runController(ctx context.Context, opts managerOpts) error {
 		os.Exit(1)
 	}
 	if err = (&controllers.TunnelReconciler{
-		Client:   mgr.GetClient(),
-		Log:      ctrl.Log.WithName("controllers").WithName("tunnel"),
-		Scheme:   mgr.GetScheme(),
-		Recorder: mgr.GetEventRecorderFor("tunnel-controller"),
+		Client:     mgr.GetClient(),
+		Log:        ctrl.Log.WithName("controllers").WithName("tunnel"),
+		Scheme:     mgr.GetScheme(),
+		Recorder:   mgr.GetEventRecorderFor("tunnel-controller"),
+		ServerAddr: opts.serverAddr,
```

Comment:
I think in a subsequent PR, i may refactor this to construct the driver here and pass it in similar to the ingress controller https://github.com/ngrok/ngrok-ingress-controller/blob/649acbaed3589e75bf560b414f53649d7261f522/main.go#L131
It looks like region isn't being plumbed through, and adding lots of options here won't be fun. I could also just create an opts struct. 

anyways, this is probably short lived